roles/wordpress-install/tasks/main.yml

```diff
@@ -37,6 +37,27 @@
   when: item.value.site_install == True and (item.value.multisite.enabled | default(False) == False)
   changed_when: "'WordPress is already installed.' not in wp_install_results.stdout"
 
+- name: Check Existing Permalink Structure
+  command: wp rewrite list --format=count
+           --allow-root
+           --url="{{ item.value.env.wp_home }}"
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/current/"
+  register: wp_permalink_check_results
+  with_dict: wordpress_sites
+  when: item.value.site_install and not item.value.multisite.enabled | default(false)
+  changed_when: False
+
+- name: Setup Permalink Structure
+  command: wp rewrite structure {{ item.value.permalink_structure | default("/%postname%/") }}
+           --allow-root
+           --url="{{ item.value.env.wp_home }}"
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/current/"
+  register: wp_permalink_results
+  with_dict: wordpress_sites
+  when: "'No rewrite rules' in wp_permalink_check_results.results.0.stderr"
```

Comment:
@ckovey Thanks for your reply and for adding the permalink setup to Trellis. I think this issue is resolved by #466 and #467. You could see if the implementation in #467 looks good to you. 

In your branch, I liked how you simplified conditionals:
- dropping the unnecessary `== True`
- using `not` instead of `== False`

If interested, you could PR those changes [here](https://github.com/roots/trellis/blob/11516d12e42355d0304eb4fb8c15be7ca8fcf6d4/roles/wordpress-install/tasks/main.yml#L38), [here](https://github.com/roots/trellis/blob/11516d12e42355d0304eb4fb8c15be7ca8fcf6d4/roles/wordpress-install/tasks/main.yml#L63) and [here](https://github.com/roots/trellis/blob/11516d12e42355d0304eb4fb8c15be7ca8fcf6d4/roles/rollback/tasks/user-release.yml#L14). I'm happy to add them if you prefer.